AccountingService/appsettings.json

```diff
@@ -5,10 +5,15 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*",
+
+  "JwtConfig": {
+    "Key": "IiwibmFtZSI6IkpvaG4gRG94cThIIoDvwdueQB468K5xDc5633seEF",
+    "Issuer": "Nikola",
```

Comment:
This was just an example. Change both Issuer and Audience to: "https://api.accounting.com"